src/service/services.aapi.service.js

```diff
@@ -0,0 +1,10 @@
+angular.module("ovh-api-services").service("OvhApiServiceAapi", function ($resource) {
+    "use strict";
+
+    return $resource("/service", {}, {
+        get: {
```

Comment:
It's more a `query` than a `get`, what do you think ?

Reply:
yep